modules/devguide/examples/BulkGet.java

```diff
@@ -46,15 +45,14 @@ protected void doWork() {
             collection.upsert(id, content);
         }
 
-        JsonObject jo = collection.get(key+"_1").contentAsObject();
+        JsonObject jo = collection.get(key + "_1").contentAsObject();
         System.out.println(jo);
         // Describe what we want to do asynchronously using RxJava Observables:
 
-
         ReactiveCollection reactiveCollection = collection.reactive();
-        Flux<Object> resultFlux = Flux.range(0, 10)
-            .map(index ->  {return key + "_" + index; }  )
-            .flatMap( k -> reactiveCollection.get(k));
+        Flux<Object> resultFlux = Flux.range(0, 10).map(index -> {
```

Comment:
this formatting is less idiomatic I think. An aligned chain of `.map` and `.flatMap` calls if much more readable for Functional Java style.